dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/interceptors/Restricted.java

```diff
@@ -0,0 +1,17 @@
+package edu.harvard.iq.dataverse.interceptors;
+
+import edu.harvard.iq.dataverse.annotations.PermissionNeeded;
+
+import javax.enterprise.util.Nonbinding;
+import javax.interceptor.InterceptorBinding;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@InterceptorBinding
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Restricted {
```

Comment:
Add some comment with `see {@link PermissionNeeded} for usage description`
And pointer to class handling this annotation